### PR TITLE
Documentation addition

### PR DESCRIPTION
**Pull request by Raistlfiren**
Added more documentation in README and removed manual install of API.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,30 @@ Currently documentation is sparse. Please help us out!
...
 Get [docker](http://docker.com).
 
+Quick overview of Docker can be found:
+[Docker Quick Start Guide](https://docs.docker.com/engine/quickstart/)
+
+First you must install and configure the database component [faf-db](https://github.com/FAForever/db)
+You can either manually install the component and follow the instructions on the Github page or run the script - init_and_wait_for_db.sh (Linux and MAC only)
+In order to run the script,, make it executable by running
+
+    chmod +x init_and_wait_for_db.sh
```
Comment (Sheeo) [inline README.md]: We should just make the file executable

File: README.md [hunk 2]
```diff
@@ -13,6 +13,30 @@ Currently documentation is sparse. Please help us out!
...
+Modify config.example.py with the correct database parameters
+
+If you used the script, then the script should be changed to the following
+
+    DATABASE = dict(
+        db=os.getenv("FAF_DB_NAME", "faf_test"),
+        user=os.getenv("FAF_DB_LOGIN", "root"),
+        password=os.getenv("FAF_DB_PASSWORD", "pbanana"),
+        host=os.getenv("DB_PORT_3306_TCP_ADDR", "127.0.0.1"),
+        port=int(os.getenv("DB_PORT_3306_TCP_PORT", "3306")))
```
Comment (Sheeo) [inline README.md]: Configuration file shouldn't need to be changed, these values are the exact same as in config.example.py?

File: README.md [hunk 3]
```diff
@@ -21,11 +45,25 @@ Run using
...
+
+    docker logs faf-api
 
-* Install Python 3.4 or later
-* Install LuaJIT (or remove it from requirements.txt and use not the api methods)
-* Install Dependencies `pip install -r requirements.txt`
-* Install MySql Server
-* Create faf database: https://github.com/FAForever/db
-* Create `config.py` (you can use `config.example.py` as template) 
+If you want to view the raw JSON on the website, then you will need to allow 'Allow-Control-Allow-Origin'' in the browser.
```
Comment (Sheeo) [inline README.md]: This should also be a part of the core

**Comment by Sheeo** [comment]
Thanks :)

File: README.md (hunk 1 above)
Comment (Raistlfiren) [inline README.md]: True, that would alleviate the issue of having to address it. I will handle that.

File: README.md (hunk 2 above)
Comment (Raistlfiren) [inline README.md]: Good call, I didn't specify the config.example.py in my instructions. I thought it was clear with the line above it - "Modify config.example.py with the correct database parameters". I will clarify.

File: README.md (hunk 1 above)
Comment (Sheeo) [inline README.md]: Already did it here https://github.com/FAForever/api/commit/9f74e4610ae34e17f8ee5c93d2bf5bb8100d0167

**Pull request closed without merging**